src/types/paginationType.ts

```diff
@@ -0,0 +1,30 @@
+import { PaginationResponse } from '.';
+
+export const defaultPaginatoinValue: PaginationResponse<any> = {
```

Comment:
```suggestion
export const defaultPaginationValue: PaginationResponse<any> = {
```

vscode에서는 오타 같은 단어에 밑줄 안쳐주나요??